athena-example/src/test/java/com/amazonaws/connectors/athena/example/ExampleMetadataHandlerTest.java

```diff
@@ -129,7 +130,8 @@ public void doListTables()
         }
 
         logger.info("doListTables - enter");
-        ListTablesRequest req = new ListTablesRequest(fakeIdentity(), "queryId", "default", "schema1");
+        ListTablesRequest req = new ListTablesRequest(fakeIdentity(), "queryId", "default",
+                "schema1", null, UNLIMITED_PAGE_SIZE_VALUE);
```

Comment:
as i said on the previous review, this tests doesn't seem to be testing anything. This should be testing actual pagination of the example handler.

Reply:
This is the ExampleHandler for the athena-example connector (not the SDK). I've updated the tests for the SDK ExampleMetadataHandler. I thought we agreed not to change the actual connectors in this iteration.

Reply:
Added pagination to athena-example.